### ISSUE TITLE
MLO WITHOUT TELEPORT

### ISSUE THREAD

**Issue by FilipeVSX** [renzuzu/renzu_housing#2]
If it is MLO without Teleport Having the option to lock and unlock the doors

**Comment by renzuzu** [comment]
Yes, we will use built in lock sysrem or you have another idea to lock it?

**Comment by FilipeVSX** [comment]
interesting!! it can be built in no problem because I'm thinking about creating several mlos of houses and it would be great to have this option

**Comment by renzuzu** [comment]
there is two way of logic for MLO
using like polyzone style and use routing buckets to support the MLO for multiple players, buying the same house.

or use conceal native to hide each other from the interiors

havent decided which one to use..

**Comment by EclipseModding** [comment]
it's important they don't listen to each other! it is possible?

**Comment by MrNewb** [comment]
Id personally suggest as close to disc-property as possible. He did it the best so far when it comes to anything free. It was just never finished. Kinda like most of his stuff =/

**Comment by renzuzu** [comment]
> it's important they don't listen to each other! it is possible?

this is possible , and already sorted out.



> Id personally suggest as close to disc-property as possible. He did it the best so far when it comes to anything free. It was just never finished. Kinda like most of his stuff =/

i havent see disc property , is it a open source?
